sniffles/pcre_internal.h

```diff
@@ -7,7 +7,7 @@
 and semantics are as close as possible to those of the Perl 5 language.
 
                        Written by Philip Hazel
-           Copyright (c) 1997-2014 University of Cambridge
+           Copyright (c) 1997-2016 University of Cambridge
```

Comment:
Is this the correct year?  2016?

Reply:
Yes, this is a verbatim copy of their file.